addons/cppcheckdata.py

```diff
@@ -1506,7 +1506,7 @@ def get_files(args):
     """Return dump_files, ctu_info_files"""
     all_files = args.dumpfile
     if args.file_list:
-        with open(args.file_list, 'rt') as f:
+        with open(args.file_list) as f:
```

Comment:
I do not like this at all. This should be explicitly specified for various reasons.

Reply:
I was considering adding `pyupgrade` to the CI but the `open()` thing still irked me.

And then I came across https://github.com/asottile/pyupgrade/issues/714.

Given the attitude of the author I will refrain from integrating that tool and would also be reluctant to accept changes which are based on it. Such behavior should be not supported...